docker/4.4.0-alpine/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM java:openjdk-8-jre-alpine
+MAINTAINER Francesco Caliumi <francesco.caliumi@gmail.com>
```

Comment:
I am not sure how does it work here.

shall it be "Apache BookKeeper" <dev@bookkeeper.apache.org> ?